kubernetes/roles/deploy-kubevirt/templates/haproxy.yaml.j2

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
```

Comment:
Can you move service to separate template, so it will look the same as an external service template?